menu-js-component/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="index.js"></script>
+    <title>Home Goods</title>
+</head>
+<body>
+    <div class="dropdown">
+        <button class="dropbtn">Seasonal</button>
+            <ul class="dropdown-content">
```

Comment:
Pls fix formatting: ul is not a child of button, so it should be on the same level as the button.